packages/webapp/src/App.tsx

```diff
@@ -53,25 +58,7 @@ const App = () => {
     }, [env, setShowInteractiveDemo]);
 
     return (
-        <MantineProvider
-            theme={{
-                globalStyles: () => ({
-                    '.transparent-code .language-json': {
-                        backgroundColor: 'transparent !important'
-                    },
-                    '.transparent-code .language-typescript': {
-                        backgroundColor: 'transparent !important'
-                    },
-                    '.break-all-words .token.string': {
-                        wordBreak: 'break-all',
-                        whiteSpace: 'normal'
-                    },
-                    '.mantine-Prism-code': {
-                        fontFamily: 'Roboto Mono'
-                    }
-                })
-            }}
-        >
+        <MantineProvider theme={theme}>
```

Comment:
I initially started using Mantine for the new components so I upgraded it, but Radix was also already used and some stuff were missing/less convenient in Mantine